### DISCUSSION TITLE
Custom branding applies to name also

### DISCUSSION THREAD

**Discussion by decipher3114** [ReVanced/revanced-discussions#383] [Question] [answered]
Since we already have `custom-branding`, I wanna request for original name in this patch as well.
Like Youtube Revanced to Youtube only
Add this if possible and if it don't create any legal issues.

**Comment by TheJeterLP** [comment, marked as answer]
Just don't use the custom branding patch.

**Reply by oSumAtrIX** [reply]
He wants the logo but not the name

**Reply by decipher3114** [reply]
No no. I want OG YT icon as well as OG name.
XD

**Reply by TheJeterLP** [reply]
Just exclude patch then

**Reply by decipher3114** [reply]
But the name is YT Revanced for non roots.

**Reply by ghost** [reply]
you need to modify microg resource patch for that
